### PR TITLE
Update stress-ng

### PR DESCRIPTION
**Pull request by kamalca**
[Stress-ng](https://github.com/ColinIanKing/stress-ng) V0.07.16 no longer works with newer version of gcc 11.2.0

### CODE REVIEW

**Review by squirrelsc — APPROVED** [review]

File: XML/Other/ReplaceableTestParameters.xml [hunk 1]
```diff
@@ -409,7 +409,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>APP_GIT_TAG</ReplaceThis>
-		<ReplaceWith>V0.07.16</ReplaceWith>
+		<ReplaceWith>V0.13.07</ReplaceWith>
```
Comment (johnsongeorge-w) [inline XML/Other/ReplaceableTestParameters.xml:412]: Why this change?

File: Testscripts/Linux/Linux-Kernel-Self-Tests.sh [hunk 2]
```diff
@@ -81,14 +81,18 @@ function install_dependencies() {
...
+                        libgcrypt11-dev libipsec-mb-dev libjudy-dev libkeyutils-dev \
+                        libsctp-dev libatomic1 zlib1g-dev libkmod-dev)
             LogMsg "Dependency package names： ${deb_packages[@]}"
             install_package "${deb_packages[@]}" >> $BUILDING_LOG 2>&1
             ;;
         centos_7 | centos_8 | redhat_7 | redhat_8)
             rpm_packages=(make git gcc flex bison clang llvm fuse libcap-ng-devel popt-devel \
                         libcap-devel glibc-devel.*i686 fuse-devel elfutils-devel \
-                        numactl-devel glibc-devel)
+                        numactl-devel libaio-devel libattr-devel libbsd-devel \
```
Comment (johnsongeorge-w) [inline Testscripts/Linux/Linux-Kernel-Self-Tests.sh:93]: This changes may not be required.

**Review by johnsongeorge-w — CHANGES_REQUESTED** [review]

**Comment by kamalca** [comment]
These changes are flawed. The new version of gcc is only applicable to internal project at this time so I will be making a pull request to internal repro for these changes. We should make a note that this update to stress-ng will be necessary for LISAv3 is we continue to use the same tool for stress testing as we should be supporting newer version of gcc in our new version of LISA.
